src/test_c_kzg_4844.c

```diff
@@ -0,0 +1,81 @@
+/* Tests for myfunctions.c, using TinyTest. */
+
+#define UNIT_TESTS
+
+#include "tinytest.h"
+#include "blst.h"
+
+#include "c_kzg_4844.h"
+#include <stdio.h>
+#include <string.h>
+#include <assert.h>
+
+KZGSettings s;
+
+void load_setup() {
+  FILE *fp;
+  C_KZG_RET ret;
+  fp = fopen("trusted_setup.txt", "rb");
+
+  ret = load_trusted_setup_file(&s, fp);
+  assert(ret == C_KZG_OK);
+
+  fclose(fp);
+}
+
+void get_32_rand_bytes(uint8_t *out) {
+  static uint64_t seed = 0;
+  seed++;
```

Comment:
what is the point of this?

Reply:
I added a few helper functions to build blobs and field elements to be used by the unittests.

That's the function that produces random data from a (built-in) seed to generate those objects.

Reply:
I mean why do we always set `seed` to `1`, or do I misunderstand what this is doing?

Reply:
I was confused thinking we always are setting `seed` to `1` but now I see the `static`.